src/helpers/getCategoryName.ts

```diff
@@ -17,6 +17,9 @@ export const getCategoryName = (category: TCategory) => {
     case "UNFCCC":
       name = "UNFCCC";
       break;
+    case "MCF":
+      name = subCategory ? subCategory.toUpperCase() : "MCF";
```

Comment:
One to catch up with Conor/Dominyka later but I have a feeling they will want the full name. Can easily circle back later though

Reply:
I will confirm in application